From c51fe0fb249db735c5b103ec99559a0254d58441 Mon Sep 17 00:00:00 2001
From: Thierry FOURNIER <tfournier@haproxy.com>
Date: Wed, 3 Jun 2015 20:12:04 +0200
Subject: [PATCH 16/18] BUG/MINOR: debug: display (null) in place of "meth"

The array which contains names of types, miss the METH entry.

[wt: should be backported to 1.5 as well]
(cherry picked from commit 4c2479e1c455e2cc46c02cfc28ea2a185f9a7747)
---
 src/sample.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/sample.c b/src/sample.c
index 87c9f60..59c80b2 100644
--- a/src/sample.c
+++ b/src/sample.c
@@ -40,6 +40,7 @@ const char *smp_to_type[SMP_TYPES] = {
 	[SMP_T_IPV6] = "ipv6",
 	[SMP_T_STR]  = "str",
 	[SMP_T_BIN]  = "bin",
+	[SMP_T_METH] = "meth",
 };
 
 /* static sample used in sample_process() when <p> is NULL */
-- 
2.3.6

